### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by szymonjanota**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,40 +31,58 @@ export default {
...
+        password: ''
+      }
+    }
+  },
+  methods: {
+    handleSuccessSubmit(values, actions) {
+
+      // Here handle form submit and then call setSubmitting()
+
+      actions.setSubmitting(false)
```
Comment (marcinlesek) [inline README.md:48]: Shouldn't be `actions.setSubmitting(true)`?

File: README.md [hunk 2]
```diff
@@ -29,40 +31,58 @@ export default {
...
+      <button @click="props.handleSubmit">
+        Submit
+      </button>
+    </div>
+  </Form>
+</template>
+```
 
-## Docs
+More example how to use it you could find [below](#examples).
```
Comment (marcinlesek) [inline README.md]: More examples below, and only H2 with `Example` word, wtf is going here? Let's remove this sentence and in Examples section let's add link to `example-app` package repository, WDYT? @szymonjanota

Comment (szymonjanota) [reply README.md]: Sure you've got a point, I missed that.

File: README.md (hunk 1 above)
Comment (szymonjanota) [reply README.md:48]: No. `Form` sets `props.isSubmitting` after it successfully validates form and before it calls your success handler - `handleSuccessSubmit`. In future I expect it to handle `isSubmitting(false)` based on return value from user handler function which when returning `Promise` could be handled automatically for user.

**Review by marcinlesek — APPROVED** [review]

**Pull request merged**